modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/AbstractExtensionObjectFactory.java

```diff
@@ -92,7 +92,8 @@ protected ParametersResolver getParametersResolver() {
   }
 
   private ParametersResolver parametersResolverFromValues(MuleContext muleContext) {
-    return ParametersResolver.fromValues(parameters, muleContext, isLazyInitMode(properties), reflectionCache, expressionManager);
+    return ParametersResolver.fromValues(parameters, muleContext, isLazyInitMode(properties), reflectionCache, expressionManager,
+                                         this.getRepresentation());
```

Comment:
what's this representation thing for? Wasn't the problem just about the wrong type of visitor being used?

Reply:
this is just for the WARN message being logged in the case the types have to be converted, to proivde infromation about where the parameter is defined.